### ISSUE TITLE
Allow users to get the Linux Bird

### ISSUE THREAD

**Issue by phil-flip** [Yellow-Dog-Man/Resonite-Issues#2745]
### Is your feature request related to a problem? Please describe.

As the native Linux build gets sundowned/put on ice, there is no way of getting the Linux badge anymore.

### Describe the solution you'd like

Check if the instance is running with proton and add the badge.

### Describe alternatives you've considered

Adding the badge with a command.

**Comment by XDelta** [comment]
In the meantime, a mod exists that allows overriding the platform your client reports. https://github.com/isovel/ResonitePlatformSpoof

**Comment by phil-flip** [comment]
Yea. I am already using the mod as a workaround. Tho that shouldn't be the default.

**Comment by Mhowser** [comment]
According to [this comment](https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/2211#issuecomment-2192540827) there already is a check in place for us proton users, I believe it would be rather simple to tack on the Linux badge at the same time based on this condition.

**Comment by 3x1t-5tyl3** [comment]
> According to [this comment](https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/2211#issuecomment-2192540827) there already is a check in place for us proton users, I believe it would be rather simple to tack on the Linux badge at the same time based on this condition.
From what I can tell it seems to be in place. And works perfectly fine.
Running arch with proton experimental.
And that was on a surface pro 9, on my desktop same spiel. Works perfectly fine..

**Comment by Frooxius** [comment]
I don't think the badge should be added in this case.

The badge is essentially an indicator of "User is running the native Linux build", since it's considered an "alternate" variant of the build. 

When you're running under Proton, you're running the same Windows version as everyone else, so the badge would not be accurate.

We could potentially add a different badge for Wine/Proton to distinguish this.

**Comment by Mhowser** [comment]
I suppose you could use Proton's logo: 

![Image](https://github.com/user-attachments/assets/c44946e6-eccf-4012-87d9-0e850e4ee6f4)

But I feel this won't be as recognizable as the Tux badge.

**Comment by Frooxius** [comment]
Hmm potentially yeah.

I think people wouldn't recognize it at first definitely (I didn't even know it had a logo), but if it gets used, then it might become a bit more? 

I'm not sure here.

**Comment by shiftyscales** [comment]
An alternative approach I could see would be to change the existing badge to a self-assignable one similar to the potato?

But if the desire is for the badge to indicate the user's current platform, e.g. Windows (Proton), Linux (native), or another alternative, e.g. mobile, then it'd probably be best to leave the existing badge as-is.

**Comment by OrionMoonclaw** [comment]
The current linux badge already gets used by people running on proton with the mod linked above, so eh?

Considering the native graphical build is kinda dead, if someone's platform says linux, and they're in VR/Desktop, they're probably not using the fairly broken linux build. Perhaps if you have some kind of reporting on the backend for statistics etc. you'd want to make sure that part says the right thing, but I don't really see how applying the badge itself for proton users would be confusing to anyone, at least until unity gets switched out probably

**Comment by phil-flip** [comment]
In my opinion: While it is the Windows Build, Linux is not the intended way of running the engine and should display the badge for debugging reasons as mentioned by Froox.

**Comment by Mhowser** [comment]
In regards to the recent announcement, I still think this should be kept as an option for proton users, either by using the proton icon or the Tux icon.

**Comment by FlameSoulis** [comment]
Agreed. Finding out you are in a Proton environment apparently is not complicated, as many games seem to just detect this. Requiring the use of mods to achieve a baseline feature is a bit out of the norm.

**Comment by Redd56** [comment]
with the splittening coming up and cyro talking about having frooxengine being a native proccess with unity under proton. i think that would fix this issue

**Comment by phil-flip** [comment]
Since the Engine is now natively runs on Linux again, the borb is back. TT closed
